pydantic/typing.py

```diff
@@ -176,6 +177,9 @@ def get_args(tp: Type[Any]) -> Tuple[Any, ...]:
 
 
 NoneType = None.__class__
+NONE_TYPES: Tuple[Any, ...] = (None, NoneType)
```

Comment:
Can we use `set()` here for faster lookups?

Reply:
Done here https://github.com/samuelcolvin/pydantic/pull/2149/commits/dd3a75c01c7e911dd21dbb1e3a0272588dea29e8